### DISCUSSION TITLE
Show the last 24 hours of consumption

### DISCUSSION THREAD

**Discussion by TioBundy** [kalkih/mini-graph-card#1122] [Q&A]
These cards can be used to show the consumption of the last 24-48h, etc, but I understand that it does so from the moment you use the adjustment or create the card with entity.

But it can be configured to show the consumption of 0.00h of each day until 11:59 p.m. of the same day and then reset it?

Thanks.

**Comment by ildar170975** [comment]
> But it can be configured to show the consumption of 0.00h of each day until 11:59 p.m. of the same day and then reset it?

The mini-graph-card does not care about a particular purpose of a sensor - "consumption", "co2 level", "processor load" etc.
It just takes data from DB for a particular sensor and process it.

Reset - should be done on a "sensor side" (by an appropriate integration), i.e. not by a card.

Show data WITHIN a period - not supported, only a `hours_to_show` option can be used - i.e. data are gathered from NOW to some past moment.
Maximum you can do - define `hours_to_show` dynamically: assume now it is 16:30, so if you need data from a midnight, you need to define `hours_to_show: 16.5` in a template.
But templates not supported in `mini-graph-card` to define an option - so consider using other cards which do support templating like `config-template-card`.
For discussing a particular solution - go to HA community.
